.gitlab-ci.yml

```diff
@@ -254,14 +252,12 @@ validate-log-intgs-builder:
       - master
   script:
     - cd .gitlab/validate-logs-intgs/
-    - docker build --tag $VALIDATE_LOG_INTGS .
-    - docker images $VALIDATE_LOG_INTGS
-    - docker push $VALIDATE_LOG_INTGS
+    - docker buildx build --tag $VALIDATE_LOG_INTGS . --push
   tags: [ "runner:docker" ]
```

Comment:
What's the reason behind not changing the tag for the log builder to `tags: [ "arch:amd64" ]`?

Reply:
nice catch, I'm an idiot